### ISSUE TITLE
Minor Diameter mixed up with pitch diameter

### ISSUE THREAD

**Issue by cannothing** [mwganson/ThreadProfile#67]
After printing an M10x1 thread with ThreadProfile Workbench, I couldn't use it because it was too narrow. I had to recut the thread with a tap because the thread was too narrow. So I compared this thread with a nut from the FastenerWorkbench
<img width=“953” alt=“Fastener” src=“https://github.com/user-attachments/assets/bb63c079-a3d7-4e05-94f1-3dc34b326d68”>

Red is the nut from the FastenerWorkbench. When comparing these two threads and using the online calculator, I found that the minor diameter is equal to the pitch diameter.  If I use the pitch diameter of the online calculator as input for minor diameter, the thread had the correct dimension.

**Comment by mwganson** [comment]
Do you mind checking a few other sizes, too?   Also, external threads.  If there is a scale that could applied to get these all accurate that would be great.  Part workbench has a scale tool now that could be applied to the threadprofile object to scale x and y, leaving z at 1.  It can be tough to measure precisely because measurement could be at an angle.  Better to create a line on the z axis and measure the radius to that and double it for the diameter.

**Comment by cannothing** [comment]
I do not whether you read my issue after I corrected the pictures or before. Anyway English is not my mother tongue. So I think, I was not able to clarify my point. Minor diameter and pitch diameter are to different values. In your workbench is Minor diameter is in the result the same value as pitch diameter. So the names in your workbench are wrong, or your workbench.

<img width="960" alt="fastener 2" src="https://github.com/user-attachments/assets/d54fcf4f-19f8-4621-9781-b482cb35a959">

The red marked values must be different.

**Comment by mwganson** [comment]
I don't believe your measurement technique is correct.  In this image I made a v thread with a minor diameter of 10 mm and a ptich of 1 mm.  I took the cross section of the sweep object in part workbench on the xz plane.  I created a sketch on the xz plane and took a link to external geometry to a point on the flat and measured that distance to the origin.  It showed to be the expected 5 mm (actually 4.99999, but close enough).
![Snip macro screenshot-7e40a7](https://github.com/user-attachments/assets/1c4de643-ac3d-46f4-af83-cb07e2bb8cae)

**Comment by cannothing** [comment]
Your  measured distance is not the basic diameter,   you measured the pitch diameter.
Minor diameter 10mm
Nominal diameter 11.057
Pitch diameter 10.433
Your measurement must be 5.2165, because you measured the half distance of the pitch diameter. So your workbench produce wrong results.

https://www.boltscience.com/pages/screw3.htm

**Comment by mwganson** [comment]
From that link:

> The minor diameter is the diameter of an imaginary cylinder that just touches the roots of an external thread and (or) the crests of an internal thread.

For the internal thread the crests (flats) are where you measure the minor diameter of the nut.  It's the smallest diameter.  The pitch diameter would be somewhere between the major and minor diameters.
